spec/views/api/admins/passwords.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "admins/passwords.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Are you anticipating examples to add to this spec or should we delete it?

Reply:
We can delete this.